activerecord/test/cases/migration/columns_test.rb

```diff
@@ -311,6 +311,15 @@ def test_change_column_default_preserves_existing_column_default_function
         assert_equal "CURRENT_TIMESTAMP", TestModel.columns_hash["edited_at"].default_function
       end
 
+      def test_change_column_default_supports_default_function_with_concatenation_operator
+        skip unless current_adapter?(:SQLite3Adapter)
```

Comment:
slightly confused about this: is this not the feature being added to SQLite3Adapter? Why is it being skipped?